docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -61,6 +62,16 @@ \subsection{Environment}
 \wss{Hardware and software}
 
 \section{Goals}
+\begin{itemize}
+\subsection{Work Orders}
+\item Shop owners can create, update, view, filter, search work orders. Any details such as estimated time, parts used, and costs related to the work order and showing the technican assigned to each order. Allow the shop owner to send work order receipts to the customer and request payment.
```

Comment:
```suggestion
\item Shop owners can create, update, view, filter, and search work orders. Any details such as estimated time, parts used, costs related to the work order and showing the technician assigned to each order. Allow the shop owner to send work order receipts to the customer and request payment.
```

I don't really get what `"Any details such as estimated time, parts used, costs related to the work order and showing the technician assigned to each order."` is trying to say. I get that these are properties of work orders, but sentence does not make sense to me.